python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -725,15 +725,15 @@ def get(self):
     # installed concurrently, leading to errors:
     # https://github.com/ray-project/ray/issues/17086
     # Now we use a global lock, so the envs are installed sequentially.
-    worker_1 = VersionWorker.options(runtime_env={"pip": ["requests==2.2.0"]}).remote(
+    worker_1 = VersionWorker.options(runtime_env={"pip": ["requests==2.22.0"]}).remote(
         key=1
     )
-    worker_2 = VersionWorker.options(runtime_env={"pip": ["requests==2.3.0"]}).remote(
+    worker_2 = VersionWorker.options(runtime_env={"pip": ["requests==2.23.0"]}).remote(
         key=2
     )
 
-    assert ray.get(worker_1.get.remote()) == (1, "2.2.0")
-    assert ray.get(worker_2.get.remote()) == (2, "2.3.0")
+    assert ray.get(worker_1.get.remote()) == (1, "2.22.0")
+    assert ray.get(worker_2.get.remote()) == (2, "2.23.0")
```

Comment:
Not really important but I'm curious about the reason for this change, did it not work with some pip version?

Reply:
cause the package azure-core 1.22.1 requires requests>=2.18.4,  and we enable pip check by default, this case will raise RuntimeEnvSetupError,  I think the better way to fix this case is to set pip_check to False, I will modify it later.